RocketChat/Rocket.Chat
# Regression: Jump to message not working

## Proposed changes (including videos or screenshots)





## Issue(s)


## Steps to test or reproduce

- Scroll up and get a message link (kebab menu) from a message that is hidden before loading more messages.
- Paste it in the url bar

Sometimes there is a race condition and the room is opened but you don't go to the desired message. When scrolling manually to a point where the message is loaded, it then scrolls and highlights the message as it should.
## Further comments

TC-486
